docs/build.md

```diff
@@ -0,0 +1,120 @@
+# Builds
+
+A Build is a resource object to schedule and run a single [Cloud Native Buildpacks](http://buildpacks.io) build.
+
+Builds provide more granular control than the [Image resource](image.md). However, you will need to manually specify the source, and the cache volume. Additionally, you will need to create a new build for every build execution as builds will not rebuild on source code and buildpack updates. 
```

Comment:
You sort of allude to it by saying that "builds provide more granular control" than the Image resource.  But calling out at least one use case in which a user would configure a build resource rather than an Image resource.  EG. to enforce specific memory requirements.

Reply:
Updated to make clearer that the granularity is around when builds are executed. (It is unrelated to memory requirements)